.github/workflows/test.yml

```diff
@@ -5,13 +5,15 @@ on: [pull_request]
 jobs:
   build:
     env:
-      RELEASE: release-py3x.yml
-
+      RELEASE_3_X: release-py3x.yml
+      REPOSITORY_3_X: repository-py3x.yml
+      RELEASE_3_12: release-py3_12.yml
+      REPOSITORY_3_12: repository-py3_12.yml
```

Comment:
numpy just serves as an example. You can probably pick any other package instead, and then keep it simpler.

Reply:
Yes, I overcomplicated it a little bit